source/draw/RasterBgr.ooc

```diff
@@ -63,13 +63,13 @@ RasterBgr: class extends RasterPacked {
 	}
 	distance: func (other: Image) -> Float {
 		result := 0.0f
-		if (!other)
+		if (!other || (this size != other size))
 			result = Float maximumValue
-//		else if (!other instanceOf?(This))
-//			FIXME
-//		else if (this size != other size)
-//			FIXME
-		else {
+		else if (!other instanceOf?(This)) {
```

Comment:
We could add one, I suppose, but if we want it to return its "native" color (i.e. RGB or YUV or what have you), we'd still need functions from comparing any color to any color, and at any rate, it's pretty much guaranteed to run much slower than this will.